dashboard/app/controllers/pd/user_admin_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Pd::UserAdminController < ApplicationController
+  authorize_resource class: :pd_user_admin
```

Comment:
`authorize! :pd_user_admin` and `authorize! class: :pd_user_admin` both raise `undefined method `authorize!' for Pd::UserAdminController:Class`.  Andrew explained to me that since there isn't an actual Pd::UserAdmin model and since `:pd_user_admin` is just an arbitrary Symbol referenced in Ability.rb, I need to invoke authorize in a specific way.  I'll look more closely at the signatures in the CanCanCan gem for authorize and authorize_resource to see if I can resolve this, but I'm leaving it as `authorize_resource class: :pd_user_admin` for the moment.